### PR TITLE
docs: new look for documentation pages

### PR DESCRIPTION
**Pull request by iisakkirotko**

### CODE REVIEW

**Comment by iisakkirotko** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/widgetti/solara/768?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#768** <a href="https://app.graphite.dev/github/pr/widgetti/solara/768?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#760** <a href="https://app.graphite.dev/github/pr/widgetti/solara/760?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @iisakkirotko and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: solara/website/components/markdown.py [hunk 1]
```diff
@@ -27,4 +27,13 @@ def MarkdownWithMetadata(content: str, unsafe_solara_execute=True):
                 solara.Meta(property=key, content=value)
             else:
                 solara.Meta(name=key, content=value)
-    solara.Markdown(content, unsafe_solara_execute=unsafe_solara_execute)
+    with solara.v.Html(
+        tag="div", style_="display: flex; flex-direction: row; gap: 15px; max-width: 90%; margin: 0 auto;", attributes={"id": "markdown-to-navigate"}
+    ):
+        solara.Markdown(content, unsafe_solara_execute=unsafe_solara_execute, style="flex-grow: 1; max-width: 100%;")
```
Comment (iisakkirotko) [inline solara/website/components/markdown.py:33]: We might want to port the `max-width: 100%;` back to the Solara component, it should remedy the mobile overflow issues Solara apps tend to have when using markdown.

**Comment by iisakkirotko** [comment]
There are still some mobile styling issues, but I think those will be much more difficult to fix neatly. For instance some wide Solara code outputs overflow (there is even another button hidden off screen in the screenshot):

image

**Comment by maartenbreddels** [comment]
I guess it is not related to this refactor and is a problem with this example itself, right? We don't allow for wrapping I guess. I'm okay with leaving this as it is.

**Comment by iisakkirotko** [comment]
### Merge activity

* **Sep 12, 6:57 AM EDT**: @iisakkirotko started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/widgetti/solara/768).

**Pull request merged**